.github/workflows/clp-package-image-build.yaml

```
@@ -0,0 +1,104 @@
+
+  ubuntu-jammy-deps-image:
+    name: "ubuntu-jammy-deps-image"
+    needs: "filter-relevant-changes"
+    runs-on: "ubuntu-24.04"
+    steps:
+      - uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683"
+        with:
+          submodules: "recursive"
+
+      - name: "Work around actions/runner-images/issues/6775"
+        run: "chown $(id -u):$(id -g) -R ."
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:7: Quote this to prevent word splitting

(shellcheck)